apps/nextjs/src/app/api/auth/[...nextauth]/route.ts

```diff
@@ -3,7 +3,7 @@ import { NextRequest, NextResponse } from "next/server";
 
 import { handlers, isSecureContext } from "@acme/auth";
 
-export const runtime = "edge";
+export const runtime = process.platform === "win32" ? "nodejs" : "edge";
```

Comment:
Pretty sure This doesn't work as this string need to be statically analyzavlr?

We can just remove it though if it's causing issues?

Reply:
it appears to work for me locally (i.e. I'm able to develop on Windows with this change) and my vercel deployments seem fine as well (although I'm not sure if they're using the correct runtime now).

As far as I can tell there are 3 options:
1. update runtime export as done in this PR
2. remove runtime export from all pages
3. leave all code as-is and just add note for Windows developers

I defer to you on the best approach moving forward though, and happy to update this PR at your discretion.

Reply:
> it appears to work for me locally (i.e. I'm able to develop on Windows with this change) and my vercel deployments seem fine as well (although I'm not sure if they're using the correct runtime now).

if this is the case, it means you're using different runtime in prod vs dev which sounds scary. I prefer no. 2

Reply:
Good point. If you think that having windows-compatibility-by-default is more important than edge-runtime-by-default, I can do no. 2. Otherwise, I'd do no. 3. Do you want me to proceed with option 2?

Reply:
yes, I was incorrect about my original solution working, my IDE just wasn't reporting the error, and the line was being ignored entirely (which meant it was functionally equivalent to deleting the line, at least for local dev, but obviously not the right solution)

Reply:
> Do you want me to proceed with option 2?

Yes

Reply:
@juliusmarminge I made the changes to this PR that we discussed. Do you think we need to update the `drizzle.config.ts` as well? I'm not particularly familiar with this, but should we switch to a pooled DB URL now that we aren't using edge runtime?

Reply:
you don't want a pooled url when running migrations/schema changes